pages/signup.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://altur-ui.netlify.app/stylesheets/main.css">
+    <link rel="stylesheet" href="../style.css">
+
+    <link rel="stylesheet" href="../stylesheets/signup.css">
+
+    <title>Altur Store | Signup</title>
```

Comment:
You should put the page name before the brand name so that when tabs get smaller, user sees the page name first (because for user, the page name is more important than the brand name because they already know which website they have opened in that tab)

Reply:
Updated it!